### DISCUSSION TITLE
Error Pulling guilds

### DISCUSSION THREAD

**Discussion by cod69420** [Tyrrrz/DiscordChatExporter#1279] [Q&A]
![image](https://github.com/user-attachments/assets/7eab51b0-6d2d-47d2-861b-aabb5a32e21f)

I get this message every time I enter my token. If anyone could help that'd be great.
P.s. sorry about the terrible account name, and I know there are other post about kid of similar things but they had problems with guilds and channels also the solutions I could find there didn't work for me.

Steps to recreate:

- Open dev tools on discord app
- Go to Application -> Local Storage -> https://discord.com
- Reload discord 
- Copy token
- Paste into exporter
- Press enter

**Comment by Bugubugbug** [comment]
I've got the same issue.

Remember not to copy the "quotes" on either end, and I'm not sure about the "=" included in mine.
But yeah, no luck.
I read on a Guide for this Exporter that it may be obsolete now and will not work due to changes.
Let me know if you figured it out,
cheers.

edit:
This method WORKED for me,
https://github.com/Tyrrrz/DiscordChatExporter/blob/master/.docs/Token-and-IDs.md

**Reply by haifaisms** [reply]
I've got the same issue.

I clicked on the method you linked & tried Using the console, Using the network monitor, and Using the storage inspector.
They all give the same token.
None work.
Which method worked for you?

I appreciate the help :)

**Reply by Bugubugbug** [reply]
Alrighty, so I couldn't remember and ended up going through all the steps again, this is how I got the token which worked;

**Using the storage inspector**
- F12
- "Application" tab up top,
- "Storage" subtab below, > Local storage > https://discord.com
- Look for "Filter" up top > type "token" (It will NOT be shown)
- F5, you'll see "token" and tokens appear for a second,

You've gotta be quick with this step;
- Hit F5 to refresh and token will appear > double-click the "token" value > copy it with CTRL+C.

**Reply by haifaisms** [reply]
I honestly didn't expect a reply! It worked! Thank you so much @Bugubugbug
